app/models/metric/container_storage.rb

```diff
@@ -0,0 +1,13 @@
+module Metric::ContainerStorage
+  def self.fill_allocated_container_storage(obj)
+    return {} if !obj.kind_of?(ContainerProject)
+
+    sum_storage = obj.persistent_volume_claims.inject(0) do |sum, volume|
+      sum + volume.capacity[:storage] if volume.capacity.present?
```

Comment:
@zakiva should this also check if the pvc is bounded or not? 
I think so since only then it would mean that the project was truely allocated a pv.

Reply:
I think there is no need to check because capacity represents the resources of the bounded volume. In case the claim is not bounded, capacity will be an empty hash.